src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -500,11 +500,18 @@ function MoneyRequestView({
                                     getErrorForField('tag', {
                                         tagListIndex: index,
                                         tagListName: name,
+                                        tagListValue: TransactionUtils.getTagForDisplay(transaction, index),
+                                        policyHasDependentTags: PolicyUtils.hasDependentTags(policy, policyTagList),
                                     })
                                         ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR
                                         : undefined
                                 }
-                                errorText={getErrorForField('tag', {tagListIndex: index, tagListName: name})}
+                                errorText={getErrorForField('tag', {
+                                    tagListIndex: index,
+                                    tagListName: name,
+                                    tagListValue: TransactionUtils.getTagForDisplay(transaction, index),
+                                    policyHasDependentTags: PolicyUtils.hasDependentTags(policy, policyTagList),
```

Comment:
I think I suggested putting this in the data before, so sorry about that, but I think we should pass this as a 3rd param, because this isn't violation data, and thus it shouldn't be defined inside TransactionViolation's data